### ISSUE TITLE
Never reach informative error message about bad ladsftp credentials

### ISSUE THREAD

**Issue by NGenetzky** [USGS-EROS/espa-surface-reflectance#39]
Script will fail with AttributeError caused by  before reaching the helpful error message and exiting gracefully.

From Commit ec47978f0e15b116dd1a08400405af873f389bc6
```
57+        print "LADSFTP username: " + self.user
+        print "LADSFTP password: " + self.password
+
+        # verify that the XMLRPC service returned valid information and
+        # the username and password were set in the configuration
62+        if len(self.user) <= 0 or len(self.password) <= 0:
+            msg = "Received invalid sized credentials for LADS FTP from " \
+                "XMLRPC service. Make sure ladsftp.username and " \
+                "ladsftp.password are set in the ESPA_XMLRPC."
+            logger.error(msg)
+            return ERROR
```

**Comment by gschmidt-usgs** [comment]
Valid catch.

--------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT, Inc.)
Contractor to the USGS Earth Resources Observation and Science (EROS) Center
Rapid City, SD  57701
605-593-2722   Phone
605-394-6078   Fax
gschmidt@usgs.gov

Adaptation - "Manage the unavoidable"
Mitigation - "Avoid the unmanageable"

**Comment by gschmidt-usgs** [comment]
This has been fixed in the current development branch and will be released in version 0.3.2.
